srtcore/queue.cpp

```diff
@@ -136,6 +136,8 @@ int CUnitQueue::init(int size, int mss, int version)
     return 0;
 }
 
+// XXX High common part detected with CUnitQueue:init.
+// Consider merging.
 int CUnitQueue::increase()
```

Comment:
"High common part detected with CUnitQueue:init." -> 
"Too many parts similar to CUnitQueue:init"

"High common part" is confusing.

Reply:
// XXX Lots of common code with CUnitQueue:init.

Better?